### DISCUSSION TITLE
A string excedeu o limite em bits

### DISCUSSION THREAD

**Discussion by Monteiroemelo** [chillerlan/php-qrcode#145] [General]
Anexo imagem do erro.

**QRCode=**'data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAQgAAAEICAYAAACj9mr/AAAAAXNSR0IArs4c6QAAHidJREFUeF7t3VFyJMcNBNDRFeR7yCHd/wyO8EF8hnHwwxH2smg9TmLRTSr3d1EFIJHIQvd0N3/59bffn48b/PvXP/9BUfzt73+QXWJ0iuXkV+00lun91O/J7qpYlAenmLVGiovup3aJX107bfdLBeI9pNocaqdFm95P/VYgzkhp46ud1mPjENRYKhAHpLRR1U6LMb2f+q1AVCA+4koFogJx5MZVYtVLjMfj1hNEUiA9sXQkS4DayCPJV09t9ZFck2s9NnwkNU+wUkFUu6S+ST0SDE5+300QG42lACRk2chDi6F5TMec4KyxTPtQrBR7tdPGV7sKhCJ/sEtIpW6V4LpfYqekn445wVljmfahWCX10OZNclMfyVQ2jUEniGlEcT8lvTYluj1ey+oJqLEkTaSxaL6JncaidhWIoBoJqdStElz3S+wqEOcbb0mzJfXQ5p3mqXJS/U5j8PIEoQSfBl7HLwVKCTltN42LEkjrpsRVnBM7jVkxTTiU4KI10viUk4q97kc3KaeLNg3eNCgKntopmRWXxE5jUUyn7aa5pg04jYvWSONLuKa5dYI4PM6thUzskgIpMdROY5lufN2vArFzOaZ86QRxeLdDwVM7bcpEhKZj0YaetqtAVCD47rqOX0pSbaJpuwqEVih7ilDvGagI6X7KU90vOSgUaeX4rSeIDaC0eRV4JZ/ul5BKfajddCxK0g2/ioHWN8ltg/caXwXiwAwlpJ4cSr5ErJS4d4pFSar1UAx0v6S+SW4ViIuu+7U5Ngg0HYs2h/rdEKukiaabV3FRnJPcKhAVCOXjQ8VKicuOg8lKY0maqAIxf7/my/3MuaGkyUmZkFQbtQLxGBdJxTSpbyJ+G7zX+G59D0KbSAHV/TbsNObEbiOPaR/avIqLxpfsl6xVEdKG1nx1vwqEIjpsp6RK7IZDXtmuAtHnIKLnIBLFXWE4OkkaX9UfQ7mVWQWiAlGBeGQkqEA4fqp+KtgbB5TGojeD9T7bl7tJuVFc9TFtl5CgAlGBeOPjbQTiuzRH0li6dtpOsU/8qo9E1NRHcomhPvRE1Wkh2U99KC4JBi9PEInTBIBpQmoTacGT+JJYpuPT+ib5qg9thOT0VPyUu8l+6kNxUZzVL/2KkTjVQLRh1E6LpkRTv9N2in3iV31UIHysTxpacda6qV0nCPweRCIuSaOqWE3HlxBI81Uf2lgJVoqfHm7JfupDcVGc1W8nCPxbn9oI03Za8MSv+tCTLWlebYTER9LQ6lfz0EZN9kvqe+u/rJUQMmmY+j0/3lxcrsNFm3zargKBlx0VHP9psUIyLyTTja/7VSAqEPzgWht/vvH14NGGnrarQFQgKhCP6xq/AoGSpkDVrqP+G6X+ajzANho3++X5fD7Hd/2JG979LnIyhuvd9cRHcjdc/eqdec1X6aTxKYc0vsRvUg/FJbGrQBzQ04JrI+hpd3dCTuOi+SrBNb4KhCL6eFQgKhDMFm1AFc4KhH8ti4s0bFiBqEAwpSoQO/c+uCALhhWICgTTrALxFxSIX3/7/afepEyuv5W5eqMnufbUcXjDh+IyHcs0BnfihmKqdgn2isuG3U9/DkKTUOCVpMl1sMZyp9w28lXstTnujp/yYBp7xWXDrgIRsEALFLjgT75Pk1RjTjBI1ibx6drETkUyEV3FL7GrQAQsUOADFxWIBLwPHqgKt6TlFQiCyW/q4HZHs96DOKOXkFTrkYhksjaJT9cmdgn2isuGHf2KoXev1U7H4Y3GVx9JzEqWjYJrjTRmbSLFWeNL6qExqw+9TEj8bqw9YV+BOLyspcVQMmuzVSD8PRNtXhWmpOYViOE/tjtdXG3A6UJWILSt/IlBxXSaQ5qJcm1amDS+xK4TxAG9pJBK5oRUiY/piSQhn+Ks+VYgkmr4PateYvQSg1+dTihZgUjQ21n78gQxHZ6eEsnJm8SsfhMferlzp1j01E5y+y5TzzQ3kv1UnE8+aIJIglNSKTGUfEnMd2rKO8WitdQaJQdFwhflhjbWVTWazqMCgYheVfAN0iME/ICWYqW5TdtpvipqiUgmsSRrVegqEIiykh63YzNtDt4wMFRSKVaa27RdAMG4SCaxJGu1lhUIRFlJj9uxmTYHbxgYKqkUK81t2i6AoALxeDx++rsYOrop0ZKCJ2sT4upYqk2p+2m+iV+NZQO/hGtJfAl3N/wmPioQ2EUJyEkTYXjHT9fr2gqEvzOkPEiwT27eTotkBQIrqcTQZlMSYHgViA+ASuqmNeoEoSxFOy0abrdipjFXIPx9Cm1AncCmT0+NrwIx3ILabMNuo+005gpEBeIzREt4pcKU+Hj5EkMbQdU/2e8zBfnRVk8JzUNjmc43IcudctN6aL4bU0VS8yQPzU3re+JkBQLfVFWQE7Lo2iQWFaaEuOpD89ATUPHT/dRu2q/uV4FIkMK1emIpmdEt/8au+2lDa/PqfglJFdOrGvUqv1rzBHvlfSeIThBHPlYg/KdPbehpwalAKPKBnSqpnnYaip7kup82tPrV/RKSKqbTjaX7qZ3WaHq/BHvl/ctvcyqBpgmp+yXkS9YqLlrcZL/ExzSZdT+trxI8sdPGVzvNbZp/6veEVQUCPxizQTRtIiVkBSL7yTXBWbFXHwn/KhCIsoI8reAY3mVfdlJckmlGxS8hc+IjyS2pr65NapRg2gmiE8TxMW1tNiW47peQOfFRgThPWxWICkQF4nFuDhU/tVPxm55g1e/xHoT8de/EgYJ3lZ3mpkWbvva8ChcdaTfi09Ndazm9n3JDJxzlkOahNTrFR89BTAOvAW/YaW5KAi3uRm6JjwqE/y0P5UYF4vDQUULSjbUViDPKFYgKxBszOkHgPQg9JTpBzMu6jtIq9tP7KTc6QXSCOHaHEne+tV7fsRNEJ4iVCeKq5tBTQk/8ZD9t02msNGb1Oy0ayX7TuekUMF3LqzBQvz/9EkPJp8CrnRKoAuEnpZIqqZHyReur+1UgzjyoQByYkVwranOoMCX7TTdRBcKrocKUYJrUV/1WICoQ/G0KJZW2UbJf0hxJfLq2AnFAKim4Aq92SiA9yZP9NGYlle6nMavf6fom+03n1kuMDy4xns/nUwn3Z3ZatOmmvIrgG6TSJkqw/7O6/uf/FWfFRfe7e26K31W1TC6ZX34XQ0mg4CVJJETTtZpHQmbFQO00ZrVLsNLmmObVRm7qQzFIODR9+FYgggelpsmsja92Sly1q0AoUme7CkTwoFRCeiWuFiihQaL+ioHaJXnoSaQ+EuwTTDU+5ZDup4fHRi0TH50gOkEw55MmqkBc93WrSwQiUXUl2lWkUkCTPJITWrFP4lMMVF00Ft0v4UbiQ9delW+Cy2ntyxOEknS6ERT46fimgZ/GZXq/CkT2ERnlqQqO8k/t9BKoAnFASptDSaBidaf9FAMluOam+yWNkPjQtVflm+DSCQKrq82hJKhA+PseWCL+TJ7upyeq7qfc0P208dVO8+0E0QmCm02FLrncmW4Y3U8bRvf7NgLx4zcppxNT4PXUni6Qkn4DF8VKMZi20xoldhqz1k33U1FTv4qBxqf80wlC7d69rKWBaGJK+jsBqmRJMNC1SkjdL7HTGiV2Gt8GLpqH8iWJWftSG1/tKhD4cJcWSAmudgmp1IfaacMkdhrLBi6aRwVCq4Z2ql4JCbSh1YfuhxCwmcbHGwaG2jCJnYa3gYvmUYHQqqFdBQKBWvqDLhqNNkxip7FUIM5IJb11qhv9ipEUQ09e9aH76b0PJaSeErqfFvKqPK6KL6lvgr3WV3FRPqvfhAeK6cvPQXzFZBNAlWgKvMai+yX1UEJqI2huG5iqD8UvmYTUh9YjwTnhVScIZdXBToHX4up+CfmUkBWI88NdiktSo2keJPtVICoQx29SaiOo+CnMSmbdL4mvE8TjUYEImJaQ+aoG7AThL2FVIA5/ek9JryNUAnLQu7xUG1XzVccbuHyX3KbzUI5rLaftNrimk9XLD0ppEhuNkBRomnwaywYu3yW36TwqEGeWHn/FePVdjAqESsHZrgLhb3hWIOa51gkCMZ0mH7odvzGoBVdh0jyuuqeR5NEJohME87sC4TftFFRtXm3U6RqpX8132k6nc/Wr+dKDUkkxNBA97RSA5BTTtUksWvCksdRHku+GD8X5Kp4m3E36Y8NvBeKiNze1sSoQKg/ZF6Lv3qiKQsIrFZx3z0FcpcyarIKnzZacqBqL5qYxq53Gpw2jeVyFqeKi+Sp+isvd/XaC6ARx5LwSVxuhAnGWFsX5KmGqQFQgKhAP/3n1qka9yi89B5Gof3J5oqAkY2RyAiouCQbJCZP4TbDXtXrNq/ttYKUxT3My2S+J+bT23ZOU2ghJIEmjJuAlfhWXpFE3SJ9gkMSnfKlAZG+RKs5aywqEMvJgNy1WWrSEBBUI/wVkGuer+KKHVicIvAehmnFVwaeJm+SrazVm3W9DTDXmDR4kTZ6s7QShjOwEcXw8PIDv+Ad7dL8KhN9s/ekCoUXT6/Rkv0TV1W8C6EZ8SR5ao+RSRONTu40TWmNRXJKYNZaEa8pxmiCSgBOlV7+a7HfZL8mjAuEnrzZggul0f2gvqF0F4lBdBS9p1GlibJBZ8522S07jjVpWIIKKX9UIid8NUiXxaTmmx2H1O21XgXBElbtq1wmiEwR/m8JpOmtZgXA8tfHVjj45pyeRpzFrqQSa9eoPtGyMoFfV6E6TkOI8zQPdb6NG2gtatwqEVvdgp8VQ4qqqJ/cbgnSPS5Void+ksTbi09ySPNSHclJxqUAo8hWICkTAlbelFYgQwFeXq2q+uv9H6xK/ulaVfoN8OglN45zkpvhNx3zVlDfNq04QATO0GNpYvcQ4F6MC4SRVTqpw0helNgi+obgKipZDiTtdNBUStUvy1dzUx0aNkph1bWKnWCX11bUVCK3Gwa4Ckf2SowdPUKLjdb82r8anzaZ2mm+yn66tQGg1KhDjzaYNGJRoPGYVl8RO89Um1+n8FHMFQqtRgRhvtgrE/Hsheomm4lKBqEAwAkoqvfSqQHwBgfjxb3Nq0ZQEyailhGSGo6Hmhtuxmaq/joxJLZO6ccJomNTjTphu1AMh5W970LsYSpYNAJKCK3gJIdWHYqX7acxay8ROY1Y7ze3umGp8ejBu4FKBCO4tKMHVLhE/JUvS+EpczVftNDdtwGm/iqnGpzhv4FKBqEBENx8TUZtuVG3Aab8ViH/+4x2mql4KXmKnBVc7zU33U7uk2TTmBGc92TRftdPcKhCKqN8cffcrhrt4bzlNoGQ/JdVXbJhTjZJ8teZJPdSH5qZimsSsmKowJbFchUsFAqejREiS5tC1SmZtrGlCah7TfpOmVEwrEFjdpBgbxJgu5HS+CPPRTMlcgfDxWjGd5pXyIOGfru0E0QlC+cg3M3lDNFQyX3XIVCAWCnlVcZNLh4S4CCmb6WnXCaITxBuplLv0qLWSSknKrEdDbXLc7lYfcf2K2Cdir8RNuKaYKl82YtZYkl54+WUtBTQpmgKQjHPqIwFZfajdV8S+AjH/GnzCF+3LCgSiXIFAoB7ZuK44q51GraKr+3WCONzI01NCQU7sNgikKpzkodPR3bHX+LRuaqfYVyDOSHWCQAZNExLdHs2UzF9RwBRntVOcFVPd71tPED++7n0VeFqMaTst7rRf3W+68bW+isvd40twToTpTjgnsdBXrRXkZLRMfCRrtRESH8nauzfg3eNT7JUHmm/SlIkwTV+mViD+/sc7TLW4Sr7ETgmpPjS36Ya5Kj71O53vnXBOYqlAVCCOPTTdMNqoCZl1bTLpqmBrLBs4J7FUICoQFYhPPFn4lxOI5/P5/DN1vzsoqpB/luf/+39V+rufTgkGybWx1mgaZ81Xc0vsNBa1077U/ehnTiX49M0QBT7xq0ApBtOk14InfhMMNmpUgfAKKV90xwqEInWwmybuVc0WQBC9o5KIWrJW89V6JHYai9pVIA73BzpBZI83K/kUZyVp0uTJWs03afzk8ND4dKpN9usEEaCXkEDXbjRbAEEniA/ePdH6JthfJhCvPkmpoKidAjCt9HpSanE1X7VLcJleqxgkdtMiqRgkMW+s1SlqOpaXf+ZUgqudFrIC4R/7SDCdJpruV4E4I1WBCG4MqghdRb5E1KabXLHShp62u6pG03lM71eBqEDw30tMmjxZO016Fb+Ny8CN3BIfFYgKRAXigycaKxD+y1UiQqe1dA8iOXU2RkY9ia5S4Wn8kjySWBKc1a/yZboRFFPNQ7HSPJL4EoGtQGiFArtpUilZlKQb+ykGFYjsJqXipzWvQASNr0u1Oe7U0NO5KQZKcI1P7bRhNA+t5UZ8nSCC+xdaoMRumlRKZiXpxn6KQQWiE8QRgYSkG41QgTgjkDS+/vybYK9rlX+ar3JyI76fPkFoEomdnhxaSI1FC36V3TTRErLcKRatR8IDXauYKseT/RLRPb6L8eqj1gl4CdEqEOcnKZN6JJgmpNdGUL7cPY8EK218tVPs6SZlQj5dq+AlJEiIpifWtJ3GrDgrMXQ/rZvup/VVnNXvRh6JD218tVMeVCDwk3NKyGm7CsS5xRXnCoRPnL3EOLBFiXaVXQWiAvGGgE4GavfyBKELp4mriV01gupJlOByVW4b4qe82hjDk1iUp8qXpObqQ/M92b27xEg22yhuAqiuTYCvQGQj7QaHlOOJcCqHlJMJLppvBQL/CLEWV+20uAlZdK0KmJ6UaqckVaym99M81E65oXVLcFGsKhAViCNvk5MyaZhkrZJeG0tjUbsKxAEBLUZS3ERxda0WV+0UF41PGzqJTxtB7ZKaax4bsSQ+FAOd8hSXxO8vP/7hHCWfkj4J7iv6mMYvIWQiONN101iS5viKPjbqm2BagcBP62vDVCCyb2YmZK5A+Eyhh28FogLxULJoA6pIKp2n47uTCHWCOFQjIZqS6iof2hwJ6ZO1G82hGGgtp/PdwEB9VCAqEPwUXHIZo2uVuNP7qWBvxHcnH99CIFTpEzsFSommp850IygGmofuN016xU/zuGqqSOqrmE5jkGCldVNc6B5EQlJdW4FQpNxOyaLYK6mmG0v3U2S0odWv7qeNr3YaX4JLBeLml0Ba3IQsFQhHebp5FftEhDS7k48KRAVi/B6JipWSXvdLGkHXViCCn/0UZB1Vp4uh8SlxdYTXfDU+tdP49BRL8khqWYHwP5yjNddavjxBJE2khFQ7bZhpO8VACa77KQmS/ZK1Sr6kHldhoH6T3JK103WrQATV0GJUIPwE1HJoo2qNdMJRv5rHtF2S7ymWCkRQIS1GBaICEdDsU0uVkyp0FYhPwf+/xlqMCkQFIqDZp5YqJysQn4L1NWMtRgWiAvEawz6/Sjn5skB8PqT/v0IDUb9643IaKPU7fYNOr41VhDS+xO90LMqNhGtJfTU+5aTup3ZJLd9dYqhTtUuKtkFmjS8hUEKMpLjTuU3nofXd4FpSX40vwU99qDhrLBUIfO4jIZAWY7q4FQhvq6S+6iXhgfqY5lAFogLBT1ImBE/ESptDfejkkuynPjS3xC6ZQisQFYgKxCf+ME3SqInAJn5HBSLZTBVXgZreT0FWDDQ+9ZucOhqLjtJao+mRNsEqWas1v8qH1jfh0GktPQehZNEkrtpPi6tk0XzVb1JcjaUCca6G1jypZeJD65twqAKB1dVCJkXDUI7jv57aShbNV2Oe3k/9JnYbMSc+Eq7pgVyBQAZpIZOiYSgVCAUqtNOaJ24SHwnXKhDDfzFLC5kUTYmmxdVYeonRSwzl3psd/fHe6VH1qgbU5lAAk+bVtRpLgqnGoj70Emjar4qk8lnz0P028FMMFPsKBP7MqSRQUmmBKhD+h3i0OaZrqftVIA7Npqe22mnDJEVTH9rkCTE0FvWhdtNCt+G3AnEWU+2F403KX3/7/akk/G87behpu1di/c8ajUV9VCD8Lc0KhE9CiTirSCp3e4nRS4zolxIlWgXimwiEnrJqp+NNonzJWp0W7qTq0/kmzau4KA+0HoqB+lUMEr+aWyK6mq/W7d0EoY2vdhqwAj/tV4umgG7El2CV5KFYTcenMWt8WiO1U79qV4E4PLeQkCpZq0VTkiakuhMxNBbFRQ8KrYfWXP12gjhfAnWCUEYe7KZJpU2pzZHsp2srEAGBPliq2G/woAIR1LcCcQZvmriJCHWCyH5pol8xkrE56D9e+l3i05NDgZkWMPWr9VA79atiMC04Gp/mO22XxFeBUPQCu6TggdtHBSI7PRPsVaySGimvNI/TfhUIRS+w00J2gvAnAbUciqleFqnfCkTwgFECcgL8tF/drwLhDwlNN2oFQll6nrY6QTh+L1tWICoQb+RJeJBciihxj5cYz+fzf97FUAVXZdbgNvwqyBsxa746MW3ErDf3lBvT9UjiS2LRtYmd8kB9KF/om5QanDrVQk77nQZPm1xPDsVPG3AD5wTTZK1ipT7UTjHVmqud9kKSx8lHBSL4GlUFwi8dksZSMUh8JI2laxO7CgTe9Jw+Paf3SwqpjXBVzEkDTq9VrJKmVJynfUwfPJpHJwj8oE1CvgpE9uxBQuZEhLTJN3zcWiCS5tDiJsVICpSs1ZjvVFyNRWuudsqD6Xok4pzErLhs1EPz0Fhe/tN72jAJCRLgE6D0xpHaXUVcJYHirHaKfcINzU1rlMSsuGjMup/ySnGmSwwNrgLhv2trIRNMExJozdUuaTbFQJutAnGumuLXCQLvS3xF4ioJtPHVrgKRNaXirAdPcnhUICoQCR+PaysQFYhxUiUqtxHMNOk31D/xsVGPjfFfJz/lkE5ld8pNMTjZvTxBKKCJnRYj8aFrKxA7b1oqmbVuG/upEG/EkghTBUJZdbCrQFQg3mihh1bSqEpTFZzErhMEVqMC4c2BkEZvN6oPbY5kv04Qit6wnar1sNuVG29Kqul7ARtCp/W40ymrMSsn75SbiuTxEuPVP72ngKrdNKAJKNNNpLkloqFCorFM203zQPdTTJUvd8dZ66a40AdjkmLoWk1Mm1cLrnaah+6XnEQbsST10NyUpMl+iY+ruJZwSOumuFQg8DmIjabUom3EokRTO415ej/FVJuyE4RWcthOiXGVqmu6SjQ9FTXfuxNX8VMe6H4VCL+5fMK+E0QniOjz+NMNPb1fBWJYIJITS1VdT9lkP12b5Dudh04Bmps2h+6n+ep0pPHpftO1TOJTobt7zPSn95RAaqdES/bTtdMFSvarQMy/Ias8SERI+ax2d4q5AjH8TcoKhI+0yQmta+/UbBUIrMZVQG2QqgJRgXjjWS8xUAx0bE4aS8fDCoR/LzKp23Q9dL8NDiWNf9XBmMRMlxhaIG1ABepOfpV8mlvSgKrNCX7qQ3HR/RJcknw1j+n6Jrgk/aZ+KxCHexDTJNggnzaWEkPtNDfdT/PQU1H9ah7T3ND41G4alwpEBUK5d7TTxkqcaFN2gvB7H1qPCkQFQrlSgfjgexAqkomAaZE6QSBSClRil4zD02t1P4SPzbQ5eMODYScIR0/5rDt+iwlimqSq9Oo32U+bIxEIJdVGvkpcxVRv5CX4qQ+NeaMeWssKBJ5YSoKEaBvE0DyuEqYKhN9H0CZPalmBqECMP9STnJQViApE9PagnoBKtOkTf3q/RP2TRr3Kr9ZNc0v4Mu1D99uYJHX66ATRCaITxAeqpA2tIqT7VSDwGwxfEVA9eRM7PWX1lND9dDpSv0l9k5iTtRpzIhoqEOpD8z3l9m0nCG3ADdJrwdVOC66NqvttYDVN+iQ3zVd9aH3VbhqrCgS+2r0hLupj+sRSMqud5pE024bQJfnqWm18tatAHC5FFJRp4ipJ1W9ilxBS16qd5lGByH6d0INCear16CUG3qRU4LVhEjttXo1Z91NSqd8N0ie5ab7qQycDtdPDUuN7+RJDHahd0hx3Ak9jUTvFb3q/ab8b8SU+rhIw5f2dhIkmCCWQ2ilQid2GuipJ1U7xm95v2u9GfImPCsT5q18nXCoQ2B2JWCVkVqHTcR3TPZppHmq3EYvil5zaieAkaxU/rUcFAn/FULIkwCcNrX6VQGqnftVO/WqTK6bTTZnsl6xV/LQeFYgKhHKqE8QH335QsUoOGV2rxRwVCHU6badJaIGSS4Lp3HQ/PU2mCaSYJn41t426KdfULsFPuZFMR4rpKZYv96f3tBgKigKvhUzstImSRtX4NBbFL9kvadSr+KL5aj0SnLUXKhD4eTkt2rRdQiolkMassajfZL8KhP+tEcVK69EJAp/W1MZK7LRonSC8YTpB+E+anSA6QbB+qVh1gjhDqvhpQRKck0uMfwPij5q07y/ESgAAAABJRU5ErkJggg==';

![Captura de tela 2022-09-09 154809](https://user-images.githubusercontent.com/86304368/189456186-c28ae611-b06e-48b9-96fb-71cdb9c8c2e6.png)

**Comment by codemasher** [comment]
That's a lot of bits (max for version 40 with ECC level L is 23648)! So where exactly is the problem? Was the above image rendered despite the error? Can you post an example of the code, including the string that produced the error? Also it would be helpful if you maybe could post in a language we both understand. Help me help you. Thanks! (the issue templates exist for a reason)

**Comment by Monteiroemelo** [comment]
Image 1 code. Image 2 exit. The code is the whatsapp qrcode. A shorter code(string) really works. How to solve?.
![1 - Code](https://user-images.githubusercontent.com/86304368/189482040-deb95591-3247-4b7c-8155-12c3eb254434.png)
![2 - Exit](https://user-images.githubusercontent.com/86304368/189482041-142bd26f-7e19-4981-99f1-3255cfbac5b6.png)

**Comment by codemasher** [comment]
Uh, that's not how it works. Do you want to *read* the QR Code? if that's the case, try the following:

```php
$result = (new QRCode)->readFromBlob(base64_decode(str_replace('data:image/png;base64,', '', $data)));

var_dump($result);
```

If you want to generate a QR Code, you need to input the string that it should contain:

```php
$data   = 'this is a simple string';
$qrcode = (new QRCode)->render($data);
```

**Comment by codemasher** [comment]
If you want to "convert" the given QR Code to a different output format provided by this library, you can use the reader result from the previous example and re-encode it like so:

```php
$qrcode = (new QRCode)->render($result->data);
```
